### PR TITLE
Paul/smal fixes2

### PR DESCRIPTION
**Pull request by paullinator**
### CHANGELOG

Does this branch warrant an entry to the CHANGELOG?

- [ ] Yes
- [ ] No

### Dependencies

 none

### Requirements

If you have made **any** visual changes to the GUI. Make sure you have:

- [x] Tested on iOS device
- [x] Tested on Android device
- [ ] Tested on small-screen device (iPod Touch)
- [ ] Tested on large-screen device (tablet)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206387496803858

### CODE REVIEW

File: src/components/scenes/TransactionDetailsScene.tsx [hunk 1]
```diff
@@ -382,7 +382,7 @@ const TransactionDetailsComponent = (props: Props) => {
       {transaction.spendTargets == null ? null : (
         <EdgeAnim enter={{ type: 'fadeInDown', distance: 60 }}>
           <CardUi4>
-            <RowUi4 rightButtonType="copy" title={lstrings.transaction_details_recipient_addresses} body={recipientsAddresses} />
+            <RowUi4 maximumHeight="large" rightButtonType="copy" title={lstrings.transaction_details_recipient_addresses} body={recipientsAddresses} />
```
Comment (Jon-edge) [inline src/components/scenes/TransactionDetailsScene.tsx:385]: This fix doesn't work for a number of addresses. The legacy sm/med/large prop
from Tile I think we should deprecate.

I think a better fix for now is to give a complex child that properly handles
printing out the addresses on individual lines.

Comment (Jon-edge) [inline src/components/scenes/TransactionDetailsScene.tsx:385]: This fix doesn't work for a number of addresses beyond 3. The legacy sm/med/large prop
from Tile I think we should deprecate, I don't think maximumHeight fits in our new design.

A more complete fix would be supplying a number of EdgeText components for
each address as a child to RowUi4.

**Review by Jon-edge — CHANGES_REQUESTED** [review]

File: src/components/scenes/TransactionDetailsScene.tsx (hunk 1 above)
Comment (paullinator) [reply src/components/scenes/TransactionDetailsScene.tsx:385]: I don’t see how this doesn’t fit in our design? I tested with 5 rows and it works fine. 

It’s also a waste to create text components for each row as the content is literally just a multi line string. That’s more code and View overhead for no reason I can see. 

Can you justify your thoughts on how this doesn’t fit our design?

Comment (Jon-edge) [reply src/components/scenes/TransactionDetailsScene.tsx:385]: Can't reproduce the issue anymore

**Pull request merged**